Automattic/wp-calypso
# READER - Add reader notifications page

This PR aims to add the notifications panel, typically viewed from the bell icon in the admin bar, into a page on Reader.
<img width="419" alt="Screenshot 2023-03-23 at 22 07 37" src="https://user-images.githubusercontent.com/5560595/227375706-02323772-b4fd-45ee-b6cf-c9d8df70afae.png">

The existing notifications are rendered in an iframe that is hosted on a different domain widgets.wp.com. There is an issue with third party cookies working and as such, on some browsers like Safari, the notifications panel does not render any notes.

The workaround is to find a home for the notifications outside of the iframe and this PR will add them to reader.

This page will be accessible at https://wordpress.com/read/notifications

This PR loads the existing Notification component and modifies the styling to work within a reader page, rather than from a panel that works from the right side of page, sliding to the left.

<img width="1475" alt="Screenshot 2023-03-23 at 22 09 41" src="https://user-images.githubusercontent.com/5560595/227376002-e72b1b1a-d8d3-4019-a8e5-49559b173cae.png">

### Testing

* Apply patch
* Run `yarn & yarn start`
* Go to http://calypso.localhost:3000/read/notifications and confirm notifications load and work as expected
